include/rocksdb/sst_file_writer.h

```diff
@@ -148,6 +148,13 @@ class SstFileWriter {
   // REQUIRES: comparator is *not* timestamp-aware.
   Status DeleteRange(const Slice& begin_key, const Slice& end_key);
 
+  // Add a range deletion tombstone to currently opened file.
+  // REQUIRES: begin_key and end_key both contain or both not contain timestamp.
```

Comment:
What happens if both begin_key and end_key contain timestamp? It looks like timestamp will still be appended to both of them?

Reply:
Yes, it should be only "not contain timestamp".

Reply:
Updated.